jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -36,48 +36,33 @@
  * @{
  */
 
-/**
- * The ecma property types must be lower than the container mask.
- */
 JERRY_STATIC_ASSERT (ECMA_PROPERTY_TYPE_MASK >= ECMA_PROPERTY_TYPE__MAX,
                      ecma_property_types_must_be_lower_than_the_container_mask);
 
-/**
- * The ecma object types must be lower than the container mask.
- */
 JERRY_STATIC_ASSERT (ECMA_OBJECT_TYPE_MASK >= ECMA_OBJECT_TYPE__MAX - 1,
                      ecma_object_types_must_be_lower_than_the_container_mask);
 
-/**
- * The ecma lexical environment types must be lower than the container mask.
- */
 JERRY_STATIC_ASSERT (ECMA_OBJECT_TYPE_MASK >= ECMA_LEXICAL_ENVIRONMENT_TYPE__MAX,
                      ecma_lexical_environment_types_must_be_lower_than_the_container_mask);
 
-/**
- * The ecma built in flag must follow the object type.
- */
 JERRY_STATIC_ASSERT (ECMA_OBJECT_TYPE_MASK + 1 == ECMA_OBJECT_FLAG_BUILT_IN_OR_LEXICAL_ENV,
                      ecma_built_in_flag_must_follow_the_object_type);
 
-/**
- * The ecma extensible flag must follow the gc visited flag.
- */
 JERRY_STATIC_ASSERT (ECMA_OBJECT_FLAG_EXTENSIBLE == (ECMA_OBJECT_FLAG_BUILT_IN_OR_LEXICAL_ENV << 1),
                      ecma_extensible_flag_must_follow_the_built_in_flag);
 
-/**
- * The ecma object ref one must follow the extensible flag.
- */
 JERRY_STATIC_ASSERT (ECMA_OBJECT_REF_ONE == (ECMA_OBJECT_FLAG_EXTENSIBLE << 1),
                      ecma_object_ref_one_must_follow_the_extensible_flag);
 
-/**
```

Comment:
Do we really want to remove these comments?

Reply:
No other file has them, since the static assert has a description inside. It is basically just a duplication.